streams/src/main/java/org/apache/kafka/streams/processor/WallclockTimestampExtractor.java

```diff
@@ -28,6 +28,13 @@
  * built-in <i>CreateTime</i> timestamp (see KIP-32: Add timestamps to Kafka message for details).
  */
 public class WallclockTimestampExtractor implements TimestampExtractor {
+
+    /**
+     * Return the current wall clock time as timestamp.
+     *
+     * @param record  a message
+     * @return        the current wall clock time
```

Comment:
"the current wall clock time, expressed in milliseconds since midnight, January 1, 1970 UTC"